src/core/processing/qgsprocessingfeedback.cpp

```diff
@@ -49,6 +49,15 @@ void QgsProcessingFeedback::reportError( const QString &error, bool )
   mTextLog.append( error + '\n' );
 }
 
+void QgsProcessingFeedback::pushWarning( const QString &warning )
+{
+  if ( mLogFeedback )
+    QgsMessageLog::logMessage( warning, tr( "Processing" ), Qgis::Warning );
+
+  mHtmlLog.append( QStringLiteral( "<span style=\"color:#b85a20;\">%1</span><br/>" ).arg( warning.toHtmlEscaped() ).replace( '\n', QLatin1String( "<br>" ) ) + QStringLiteral( "<br/>" ) );
```

Comment:
should we make this bold too?

Reply:
No strong feeling. Whatever we decide (@alexbruy , feel free to chip in on this particular point) , we'll have to harmonize the error messages too.

Reply:
Agreed re the error message too.

Reply:
No strong feeling either, probably using color is enough.

Reply:
@alexbruy , beyond the color, are you OK with the PR implementing a warning message concept for processing feedback?

Reply:
@nirvn I think it is good move and warning messages can be useful in some cases.